src/main/resources/files/readme.md.tpl

```diff
@@ -22,10 +22,17 @@ unit and integration tests are run on pressing Enter in the command terminal; yo
     mvn liberty:dev
 [/]
 
-
+[# th:unless="${mp_rest_client}"]
```

Comment:
Rest Client is not the only one that results in the creation of the secondary project.

I have created #356 so that we have an additional variable that indicates if there are 2 project or only 1 created (`secondary_project`).  Once that PR is merged, this can be updated.

Reply:
I've applied this suggested change.